devel/tig/Portfile

```diff
@@ -33,17 +28,19 @@ license_noconflict  git asciidoc
 # -E, -S, -save-temps and -M options are not allowed with multiple -arch flags
 universal_variant   no
 
-# We need to create the configure file,
-# the github version does not contain it.
-pre-configure {
-    system -W ${worksrcpath} "make configure"
+destroot.target-append install-doc-man
+
+variant doc description {Install HTML documentation} {
+    depends_build-append port:asciidoc
+    depends_build-append port:xmlto
+
+    build.target-append doc-html
+    # for some reason, the XML tools fail to find and use ${prefix}/etc/catalog,
+    # where these paths are configured
+    build.target-append XMLTO=\"${prefix}/bin/xmlto \
+        --searchpath ${prefix}/share/xml/docbook/4.5/ \
+        --searchpath ${prefix}/share/xsl/docbook-xsl-nons/manpages\"
```

Comment:
Can you elaborate on why this was necessary for you? It works just fine without this block on my machine.

Reply:
Oh, I did not try without. That workaround (added by you) dates back to 2013 and I assumed  it was still necessary. Should I remove it then?

Reply:
Oh, yeah, I guess my former self didn't know what it was doing. If you have the time, please remove it and I'll merge once the CI is done.

Reply:
Sure, done.